encord/dataset.py

```diff
@@ -287,6 +288,89 @@ def add_private_data_to_dataset(
         """
         return self._client.add_private_data_to_dataset(integration_id, private_files, ignore_errors)
 
+    def add_private_data_to_dataset_start(
+        self,
+        integration_id: str,
+        private_files: Union[str, Dict, Path, TextIO],
+        ignore_errors: bool = False,
+    ) -> str:
+        """
+        Append data hosted on private clouds to existing dataset.
+
+        This method is only initialising upload job in Encord Backend.
+        Once upload_job_id is returned from this method one can:
+
+         - Forget about next steps, exit terminal,
+           job will continue running in Encord BE without any interruption.
+
+         - Fetch job status with add_private_data_to_dataset_get_result,
+           this even could be done in different python session.
+
+        Args:
+            integration_id: str
+                EntityId of the cloud integration to be used when accessing those files
+            private_files:
+                A str path or Path object to a json file, json str or python dictionary of the files you wish to add
+            ignore_errors: bool, optional
+                Ignore individual errors when trying to access the specified files
+        Returns:
+            upload_job_id: str
+                UUID Identifier of upload job. This id enables user to track job progress, via SDK or web app
+        """
+        return self._client.add_private_data_to_dataset_start(integration_id, private_files, ignore_errors)
+
+    def add_private_data_to_dataset_get_result(
+        self,
+        upload_job_id: str,
+        timeout_seconds: int = 7 * 24 * 60 * 60,  # 7 days
+    ) -> DatasetDataLongPolling:
+        """
+        Fetch data upload status, perform long polling process for timeout_seconds.
+
+        Args:
+            upload_job_id: str
+                UUID Identifier of upload job. This id enables user to track job progress, via SDK or web app.
+            timeout_seconds: int, with default
+                Number of seconds method will be waiting and checking for response.
+                If timeout_seconds == 0, only one checking request is performed, returning immediately.
+        Returns:
+            Response(DatasetDataLongPolling)
+                Status of upload job.
+
+                Upload job always has a status, it is one of:
+                    - LongPollingStatus.PENDING
+                        Job will automatically start soon (waiting in queue) or already started processing.
+                    - LongPollingStatus.DONE
```

Comment:
Usually we put these explanations as part of the dataclass itself under the specific enum options. You can find examples somewhere in the SDK for sure.

Reply:
Thanks, I have fixed this.